src/ui/public/vis/__tests__/map/ems_client.js

```diff
@@ -79,7 +79,7 @@ describe('ems_client', () => {
   it('.getFileLayers', async () => {
     const emsClient = getEMSClient();
     const layers = await emsClient.getFileLayers();
-    expect(layers.length).to.be(18);
```

Comment:
Why did this change?

Reply:
EMS client is still part of the kibana source-code in 6.8 branch, and not an external package. The unit-tests for the EMS-client and the service-settings use the same test-stubs. So by adding a new layer to the test-stub, we need to fix the assertion in both locations.

For 7.8 and up, the EMS-client is an external npm-package.